src/components/AssetAccountDetails.tsx

```diff
@@ -33,16 +37,40 @@ const contentPaddingY = { base: 0, md: 8 }
 export const AssetAccountDetails = ({ assetId, accountId }: AssetDetailsProps) => {
   const marketData = useAppSelector(state => selectMarketDataByAssetIdUserCurrency(state, assetId))
   const assetIds = useMemo(() => [assetId], [assetId])
+  const fetchAndUpsertCustomTokenMarketData = useFetchAndUpsertCustomTokenMarketData()
+  fetchAndUpsertCustomTokenMarketData(assetId)
```

Comment:
worth attempting fetch and upsert from `Asset` to try and have the market data there and ready for when this renders?

Reply:
Removed this logic in https://github.com/shapeshift/web/pull/7370/commits/dedae61a38016ecb062e2d0350b53e20254d0f21 and will instead use relay on the market data service.